Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -13,13 +13,10 @@ import Utility
 import PackageModel
 
 extension CModule {
-    
-    public var moduleMap: String {
-        return "module.modulemap"
-    }
+    public static let moduleMapFilename = "module.modulemap"
```

Comment:
Great.  I had changed this a couple of times in my various passes through the code, but pulled it out in subsequent diff reduction efforts.  It's a good cleanup, I think (and especially adding the suffix "Filename" — this is the name of the module map, it isn't the module map itself).  Nice cleanup.